src/Cms/FileModifications.php

```diff
@@ -191,6 +191,13 @@ public function thumb($options = null)
             return $this;
         }
 
+        // fallback to global config options
```

Comment:
I'm unsure myself if this is at the right location.
First I had it only in the `thumb` core component, but then the `file::version` component creates a wrong filename with wrong extension. So we somehow need to pass that option to `file::version` as well.